db/migrate/20220420204027_create_songs.rb

```diff
@@ -0,0 +1,11 @@
+class CreateSongs < ActiveRecord::Migration[7.0]
+  def change
+    create_table :songs do |t|
+      t.string :title
+      t.string :artist
```

Comment:
We don't have to address this now, but remind me to talk about database level settings that can help us protect against `nil`/`empty` fields.